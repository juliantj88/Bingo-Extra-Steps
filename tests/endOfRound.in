1
test1
test2
1 R 5
2 U 1
5 R 5
4 U 5
0 B 2
0 U 2
3 B 1
0 Y 4
0 L 3
save endOfRound
4