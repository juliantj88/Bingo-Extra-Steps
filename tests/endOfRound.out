test1
test2
2
1
1
F...............
RYYY
RRRB
RLLU
BBYU
RBYU
Byrul
lByru
ulbyr
rulby
yRulb
byrUl
lbyru
ulbyr
rulbY
yrulb
.
..
LL.
....
.....
.
U.
...
....
UUU..
       
       

RRLBBYRLLUYBURRBLRYRBLBUUBRLBLULYRLYLLUYYULLLRLUYBLBYYYRBUURUBBYUBYR
0